### ISSUE TITLE
Massive of failed to upload tracks

### ISSUE THREAD

**Issue by BCafGun** [Poniverse/Pony.fm#136]
It seems, that many last tracks on pony.fm are failed to upload (you can see all of them by zero-listening statistic). Including the my last one (my profile just in case: https://pony.fm/night-blaze). There are all just frozen on processing stage.

Also, the mail of logic@pony.fm is unavailable (seems not exist anymore). Please update it for incomig bug-reports and information on website. 

Thanks and sorry for my bad english.

**Comment by omnicons** [comment]
Yep, we're aware of the issue and investigated during a bit of free time the other night. Once conventions are over and staff are freed up next week we'll have it up and running including changing the contact emails. Until then please refer to mercury@poniverse.net for the contact!

I do apologize that it took a bit to notice the issue and that it's taken this long to fix.
